internal/display/statusline.go

```diff
@@ -47,6 +47,12 @@ var statusInfo = map[string]func(*buffer.Buffer) string{
 		}
 		return ""
 	},
+	"overwrite": func(b *buffer.Buffer) string {
+		if b.OverwriteModeActive {
+			return "[>]"
```

Comment:
Looks cute, but is there a chance anyone will guess what it means?

Also, add a space after it, similarly to `$(modified)` (and for the same reason)?